### PR TITLE
Fixes Memory usage and frame-reader errors. 

### PR DESCRIPTION
**Pull request by MankaranSingh**
This PR does following changes:

 - Make lane lines and path thinner in plot
 - Load raw_images on the fly rather than loading all of them at once for less memory usage
 - Remove OPs frame reader dependency and use OpenCV for frame reading instead.  (#3)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -57,35 +54,35 @@ def frames_to_tensor(frames):
 state = np.zeros((1,512))
 desire = np.zeros((1,8))
 
+cap = cv2.VideoCapture(camerafile)
```
Comment (littlemountainman) [inline main.py:57]: That is duplicate as it is already defined in line 24.

File: main.py [hunk 2]
```diff
@@ -57,35 +54,35 @@ def frames_to_tensor(frames):
...
-  plt.scatter(parsed["lll"], range(0,192), c="b")
+  plt.plot(parsed["lll"][0], range(0,192), "b-", linewidth=1)
   # rll = right lane line
-  plt.scatter(parsed["rll"], range(0, 192), c="r")
+  plt.plot(parsed["rll"][0], range(0, 192), "r-", linewidth=1)
   # path = path cool isn't it ?
-  plt.scatter(parsed["path"], range(0, 192), c="g")
-  print(np.array(pose[0,:3]).shape)
+  plt.plot(parsed["path"][0], range(0, 192), "g-", linewidth=1)
+  #print(np.array(pose[0,:3]).shape)
```
Comment (littlemountainman) [inline main.py:79]: The pose can be thrown out.

**Review by littlemountainman — CHANGES_REQUESTED** [review]
Looks good, I commented on two minor things, basically duplicate definitions and throwing out pose, then it can be merged

File: main.py (hunk 1 above)
Comment (MankaranSingh) [reply main.py:57]: Actually this was done to play the video from start because VideoCapture ended while reading frames into 'imgs' array.

Comment (littlemountainman) [reply main.py:57]: Alright, merged

**Comment by littlemountainman** [comment]
Thanks for your amazing work, I really appreciate it !

**Comment by MankaranSingh** [comment]
No problem, but the real work is this repo by you itself !

**Pull request merged**